src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contact in the address book. Which also has subclasses Address, Email, and Phone. 
```

Comment:
No need to mention subclasses in the header comment of the super class. The superclass can be used by many other subclasses. Your header comment gets outdated as soon as it is extended by any other class